Audio.lib

```diff
@@ -1651,6 +1651,54 @@ X NC J9 -300 -900 100 U 50 50 1 1 N N
 ENDDRAW
 ENDDEF
 #
+# TPA3126D2
+#
+DEF TPA3126D2 U 0 20 Y Y 1 F N
+F0 "U" 400 1350 50 H V C CNN
+F1 "TPA3126D2" 400 1250 50 H V C CNN
+F2 "Package_SO:HTSSOP-32-1EP_6.1x11mm_P0.65mm_EP5.2x11mm_Mask4.11x4.36mm" 0 0 50 H I C CNN
+F3 "" -150 450 50 H I C CNN
+$FPLIST
+ HTSSOP*32*1EP*6.1x11mm*P0.65mm*EP5.2x11mm*Mask4.11x4.36mm*
```

Comment:
And also not include the EP size. We decided to leave it off since the user may choose to use a bigger or smaller one and we will preserve the flexibility of CvPcb. For this specific symbol though, there is no EP or mask exposed below the part.